### PR TITLE
update copyright

### PR DESCRIPTION
**Pull request by mgeplf**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -371,7 +372,7 @@ libsonata is distributed under the terms of the GNU Lesser General Public Licens
 unless noted otherwise, for example, for external dependencies.
 Refer to `COPYING.LESSER` and `COPYING` files for details.
 
-Copyright (C) 2018-2020, Blue Brain Project/EPFL and contributors.
+Copyright (c) 2018-2020 Blue Brain Project/EPFL
```
Comment (pramodk) [inline README.rst:375]: Changing copyright would require `+2` from all contributors! 😅

**Review by pramodk — APPROVED** [review]
LGTM

File: README.rst (hunk 1 above)
Comment (mgeplf) [reply README.rst]: @alex4200 what is the guidance on this?

**Review by alex4200 — APPROVED** [review]

**Comment by ASVG** [comment]
Hi Mike, 
I believe it is an joint effort with AIBS? Let me double check on the contract first. Thank you. A

**Comment by mgeplf** [comment]
This is an implementation of a spec that was done completely by us, not sure what to do in that case.

File: README.rst (hunk 1 above)
Comment (jdcourcol) [reply README.rst]: from what I can see, all the past committers are part of BBP. The IP does not belong to them but BBP/EPFL.
So you can remove that "and contributors."

**Comment by jdcourcol** [comment]
libsonata was not implemented jointly with AIBS.

**Pull request merged**